src/color-picker/color-picker.scss

```diff
@@ -105,7 +125,33 @@ $iui-focus-box-shadow: 0 0 1px $iui-xxs + 1 rgba(t(iui-color-foreground-primary-
   }
 }
 
-@mixin iui-saved-colors {
+@mixin iui-color-input {
+  display: flex;
+  align-items: center;
+
+  .iui-button {
+    margin-right: $iui-xs;
+  }
+
+  &-fields {
+    display: flex;
+    width: 100%;
+
+    .iui-input:not(:last-child) {
+      margin-right: $iui-xxs;
+    }
+
+    input[type=number] {
+      appearance: textfield;
+
+      &::-webkit-inner-spin-button {
+        appearance: none;
+      }
+    }
+  }
+}
+
+@mixin iui-color-picker-section-label {
   margin-top: $iui-baseline;
```

Comment:
Can we remove this top margin? I would like to have the option to have this label on the basic color picker too
![image](https://user-images.githubusercontent.com/88331924/137363770-ab5b9088-5cbe-40b4-8112-6158c45be37d.png)

Reply:
Oh shoot, nevermind, I don't like this:
![Screen Shot 2021-10-14 at 2 43 55 PM](https://user-images.githubusercontent.com/849817/137377260-216366ff-e348-4640-9305-153e15c8489a.png)